.gitignore

```diff
@@ -3,3 +3,6 @@
 
 # PyCharm
 .idea
+
+# Node
+node_modules/*
```

Comment:
Let's do this under `web` folder? It will be more specific to the frontend project.